snippets/settings.py

```diff
@@ -44,6 +43,7 @@
     'smuggler',
     'django_filters',
     'django_ace',
+    'product_details',
```

Comment:
I see django-redis installed but don't see any updates to settings for that. Is that missing or all just handled via the env?